app/controllers/user/dashboard_controller.rb

```diff
@@ -1,3 +1,7 @@
 class User::DashboardController < ApplicationController
-  def show; end
+  def show
+    cu_id = current_user.id
```

Comment:
I did this to get the lines shorter.  We can refactor that if it's too obscure of an object.